pennylane/fourier/utils.py

```diff
@@ -42,3 +42,73 @@ def to_dict(coeffs):
     nvecs = product(*n_ranges)
 
     return {nvec: coeffs[nvec] for nvec in nvecs}
+
+
+def _get_spectrum(op, decimals):
```

Comment:
Could you please note whether anything been changed in these functions, or if it is just that they have been moved as-is into the separate utils file?

Reply:
In general, `_get_spectrum` and `_join_spectra` now work with `set`s instead of `list`s and only non-negative frequencies are considered. This reduces the number of frequencies that are carried around and implements uniqueness in the data structure directly. 
The frequencies are sorted and negative frequencies are added in `simple_spectrum` and `advanced_spectrum` at the very end of iterating over the tape operations.
I will leave detailed change comments in this file for smaller changes.

Reply:
Ah yes!